public/app/plugins/panel/dashlist/module.tsx

```diff
@@ -0,0 +1,71 @@
+import _ from 'lodash';
+import { PanelModel, PanelPlugin } from '@grafana/data';
+import { DashList } from './DashList';
+import { DashListOptions } from './types';
+import { FolderPicker } from 'app/core/components/Select/FolderPicker';
+import React from 'react';
+import { TagsInput } from '@grafana/ui';
+
+export const plugin = new PanelPlugin<DashListOptions>(DashList)
+  .setPanelOptions(builder => {
+    builder
+      .addBooleanSwitch({
+        path: 'showStarred',
+        name: 'Starred',
+        defaultValue: true,
+      })
+      .addBooleanSwitch({
+        path: 'showRecentlyViewed',
+        name: 'Recently viewed',
+        defaultValue: false,
+      })
+      .addBooleanSwitch({
+        path: 'showSearch',
+        name: 'Search',
+        defaultValue: false,
+      })
+      .addBooleanSwitch({
+        path: 'showHeadings',
+        name: 'Show headings',
+        defaultValue: true,
+      })
+      .addNumberInput({
+        path: 'maxItems',
+        name: 'Max items',
+        defaultValue: 10,
+      })
+      .addTextInput({
+        path: 'query',
+        name: 'Query',
+        defaultValue: '',
+      })
+      .addCustomEditor({
+        path: 'folderId',
+        name: 'Folder',
+        id: 'folder-picker',
+        defaultValue: null,
+        editor: props => {
+          return <FolderPicker initialTitle="All" enableReset={true} onChange={({ id }) => props.onChange(id)} />;
```

Comment:
```suggestion
          return <FolderPicker initialTitle="All" enableReset={true} onChange={({ id }) => props.onChange(id)} useNewForms />;
```

Reply:
This will make the picker look consistent with other fields:
![Screenshot 2020-11-11 at 9 35 42](https://user-images.githubusercontent.com/8878045/98783093-a82caf00-2401-11eb-80b5-38d2a7ba6170.png)